metpy/calc/tests/test_tools.py

```diff
@@ -39,3 +40,65 @@ def test_find_intersections():
                       [238.84, 1794.53]])
 
     assert_array_almost_equal(truth, find_intersections(x, y1, y2), 2)
+
+
+def test_find_intersections_all():
```

Comment:
So let's reduce the duplication of these tests. Either:

1. Turn the input data (`x`, `y1`, `y2`) into a fixture (see [here](https://github.com/Unidata/MetPy/blob/master/metpy/gridding/tests/test_interpolation.py#L26))
2. Turn them into a single parameterized test, where you parameterize over the direction and the truth values (see [here](https://github.com/Unidata/MetPy/blob/master/metpy/gridding/tests/test_gridding_functions.py#L103), but no need to read data from a file)

I lean towards the second choice.